src/components/LoginContainer.js

```diff
@@ -27,26 +37,17 @@ class LoginContainer extends Component {
     }
 
     _email() {
-        return (i => {
-            this.email = i;
+        return (ref => {
+            this.email = ref;
         });
     }
 
     _password() {
-        return (i => {
-            this.password = i;
+        return (ref => {
```

Comment:
I had no clue what was happening lol, didn't know what `i` was and thought I'd change it for REFerence 😅